spec/grammar.md

```diff
@@ -128,12 +128,12 @@ The grammar of the language in Extended Backus-Naur Form (EBNF).
 
 ## Notes
 
-An `expression` is used in a situation where an expression is required.
-This allows the parser to short-circuit if something is definitiely not an expression where it should be.
-However we cannot always know in advance whether something is an expression, e.g. when it is a function call. 
-Those cases should be verified by the type checker.
-An `expr-or-stmt` may be used when it does not matter whether something is an expression or statement, such as the body of a loop.
+An `expression` is used in a situation where an expression is required. This allows the parser to short-circuit if
```

Comment:
Okay need to double check what tool exactly is formatting these markdown files.
I'd prefer to have each sentence start on a new line.
And whatever tool it is, a config file where I can specify the above would be great.

Reply:
It's Intellij, but looks like there [isn't](https://youtrack.jetbrains.com/issue/IDEA-257108) an option yet to disable this automatically.